src/components/Page404/Page404.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { FunctionComponent } from "react";
+import wavesBackground from "../../static/images/common/wave-lines.png";
+import page404 from "../../static/images/Page404/Page404.png";
+
+export const Page404: FunctionComponent = () => {
+  return (
+    <div
+      className="w-full h-4/5 bg-cerulean-50 bg-no-repeat bg-cover"
```

Comment:
because if there is no height it will just take the height of the image. but from the design there seems to be some space between the bottom of the image and the top of the footer.